equivalences.csv

```diff
@@ -2068,6 +2068,30 @@ RE,R-HSA-936779,ATP1A
 RE,R-HSA-936820,ATP1B
 RE,R-HSA-976075,E3_Ub_ligase
 RE,R-HSA-996770,GATA
+SIGNOR,SIGNOR-C1,NFY
+SIGNOR,SIGNOR-C2,MTORC2
+SIGNOR,SIGNOR-C3,MTORC1
+SIGNOR,SIGNOR-C14,IKK_complex
+SIGNOR,SIGNOR-C15,AMPK
+SIGNOR,SIGNOR-C98,Gamma_secretase
+SIGNOR,SIGNOR-C150,APC_C
+SIGNOR,SIGNOR-C156,PI3K
+SIGNOR,SIGNOR-C160,ITGA2B1
+SIGNOR,SIGNOR-C163,FNR
+SIGNOR,SIGNOR-C169,LFA_1
+SIGNOR,SIGNOR-C170,MAC_1
+SIGNOR,SIGNOR-C209,Hemoglobin
+SIGNOR,SIGNOR-C217,DGC
+SIGNOR,SIGNOR-C243,IFNAR
+SIGNOR,SIGNOR-C245,Adaptor_protein_II
+SIGNOR,SIGNOR-C247,Adaptor_protein_III
+SIGNOR,SIGNOR-C248,Adaptor_protein_I
+SIGNOR,SIGNOR-C262,DNA_polymerase_alpha
+SIGNOR,SIGNOR-C264,ATP_synthase
+SIGNOR,SIGNOR-C266,MRPS
+SIGNOR,SIGNOR-C304,Cohesin
+SIGNOR,SIGNOR-C313,MAC
+SIGNOR,SIGNOR-C317,Fibrin

```

Comment:
Are you doing these by hand?

Reply:
If so, maybe we should add SIGNOR's names to PyOBO (https://github.com/pyobo/pyobo/issues/107) then do some curation a la Biomappings style

Reply:
Yeah, I did them basically by hand. I got a dump of the SIGNOR complexes and scanned through it for things that looked familiar. I may have missed a few